apps/communications/dialer/test/marionette/lib/dialer.js

```diff
@@ -63,6 +79,14 @@ Dialer.prototype = {
     this.launch();
   },
 
+  
+  goToContactList: function() {
+    this.contactListButton.tap();
+    var Contacts = fromApp('contacts').require('lib/contacts');
```

Comment:
Nit: `getAppClass('contacts')`. Now that this function is working properly :smiley: